### ISSUE TITLE
Redocking experiment 

### ISSUE THREAD

**Issue by sherifelsabbagh** [StructuralGenomicsConsortium/CNP14-WDR91#7]
Hi, 

Just out of curiosity, I performed redocking analysis to validate possible docking experiments. 

The detailed explanation of what I did is mentioned [here](https://docs.google.com/document/d/1EedXmFXxe77hy1WxDA5jPITkM3joFKliXVAIKZBu7FA/edit?usp=sharing).

In brief, by using **SMINA** vinardo scoring function, I was able to reproduce the experimental pose of the bound ligand showing both low RMSD and same interactions with the key residues. 
As stated in the issue [here](https://github.com/StructuralGenomicsConsortium/CNP14-WDR91/issues/3), the ligand forms halogen bond with CYS, ALA and LEU. However I could only see a halogen bond with CYS503. But by visualising the ligand complex, I see that they are really close to the halogen atom. 

![redocking](https://github.com/StructuralGenomicsConsortium/CNP14-WDR91/assets/60114574/dd531854-f90d-44eb-9d35-9a6953c90007)

I hope this could be useful by any means.

**Comment by sherifelsabbagh** [comment]
By comparing my results with that reported in this issue #3, I can see that results are somewhat different. 

Hit -> Kd :6 ,Docking score (here) : -7.4, Docking score (with Mohamed): -9.2

Initial (with Cl)-> Kd :0 ,Docking score (here) : -7.4, Docking score (with Mohamed): -7.0

Bought (with CF3) ->  Kd :5.7 ,Docking score (here) : -8.1, Docking score (with Mohamed): -8.2

From experimental data, it appears that the hit has the better binding followed by the bought compound (with CF3). This can be correlated easily with Mohamed docking results. My results showed that both Hit and initial compound have same affinity and the bought compound has the highest affinity which is not related to the experimental data.

Personally, I do not have explanation for this, although redocking results were reliable with me.

@mattodd 
[Mohamadshahir15](https://github.com/Mohamadshahir15)

**Comment by mattodd** [comment]
@sherifelsabbagh software/approach different?

**Comment by sherifelsabbagh** [comment]
@mattodd yes I think so. But this drives the question of whether I should continue with my approach (aka software) or use his approach ?.

**Comment by sherifelsabbagh** [comment]
sorry for asking too much but I want to take this chance to learn from all experts here and help with all I can..

**Comment by mattodd** [comment]
@sherifelsabbagh  Try both (multiple) and see which gives best match with all of the observed matched pairs in the deck. There's multiple ways of doing this, no one of them is "the right way". The data need to try to match experiment.
